xml/ha_qdevice-qnetd.xml

```diff
@@ -14,8 +14,7 @@
    <info>
       <abstract>
          <para>
-            &qdevice; and &qnet; is a tandem to participate in quorum calculation
-            and distribution of votes.
+            &qdevice; and &qnet; is a software that calculates quorum votes.
```

Comment:
I'm not sure, but I think the software is uncountable.

Reply:
At least it's not corosync-qdevice/corosync-qnetd that does the calculation.

Reply:
Thanks to Alexsei and Yan. I've corrected it now and changed it to:

> &qdevice; and &qnet; participates in quorum decisions.

However, it looks a bit thin for an abstract of this chapter. :wink:  Could we add a little bit more flesh to it?

> A QDevice acts as a third-party arbitrator and can sustain more node failures than standard quorum rules allows.
> The combination of QDevice and QNetd is recommended with even number of
> nodes and highly recommended for two-node clusters.

What do you think?

Reply:
It says "third-party" in the manpage only because it's open protocol, meaning anyone could develop their own third-party arbitrators following the protocol. So far, and from the official, we only have corosync-qnetd.

How about:

With the assistance from the arbitrator corosync-qnetd,  corosync-qdevice provides configured number of votes, so that it allows a cluster to sustain more node failures than standard quorum rules allow. It's recommended for clusters with an even number of nodes and highly recommended for 2 node clusters to deploy corosync-qdevice and corosync-qnetd.

?

Reply:
Yes, I take your suggestion. Thanks! :+1: